docs/jwst/user_documentation/reference_files_crds.rst

```diff
@@ -121,6 +121,41 @@ To setup to use the server, use the following settings:
     export CRDS_PATH=$HOME/crds_cache/
     export CRDS_SERVER_URL=https://jwst-crds.stsci.edu
 
+CRDS Cache Configuration for Developers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For most pipeline users, the above settings will suffice for establishing a consistent
+local cache.  For pipeline developers or testers, however, it is important to be aware
+that if you need to switch between CRDS servers (e.g. the `ops` and `test` servers), you
+will need to establish a separate cache for each server.  Using the same cache for
+more than one server will lead to a corrupted local cache.
+
+For example, the recommended configuration for developers while using the `ops` server is :
+
+::
+
+    export CRDS_PATH=$HOME/crds_cache/jwst_ops
+    export CRDS_SERVER_URL=https://jwst-crds.stsci.edu
+
+and while using the `test` server:
+
+::
+
+    export CRDS_PATH=$HOME/crds_cache/jwst_test
+    export CRDS_SERVER_URL=https://jwst-test-crds.stsci.edu
+
+If your cache does become corrupted, the best way to fix it is simply to remove
+the local cache and allow subsequent pipeline runs to repopulate it as needed.
+For example:
+
+::
+
+    rm -rf $CRDS_PATH/*
```

Comment:
```suggestion
    rm -rf $CRDS_PATH
```
to avoid deleting every file on your system of `CRDS_PATH` is not defined.

Reply:
The removal of the `*` is fine; the subfolder will get recreated.

And on that note, may actually want to remove the `-f` also; that should be done only if one is very sure of that operation also.

Reply:
Thanks for the catch, fixed now!